redirects.yaml

```diff
@@ -136,6 +136,7 @@ certified/iot/models/?: "/certified/iot"
 certified/soc/models/?: "/certified/socs"
 certified/why-certified/?: "/certified/why-certify"
 certified/devices/?: "/certified/iot"
+certified/vendors/nVidia/?: "/certified/vendors/NVIDIA"
```

Comment:
Is this necessary? URLs should always be lowercased?

Reply:
It was included in the ticket instructions 
![image](https://github.com/canonical/ubuntu.com/assets/17607612/2811267d-5754-469d-9daa-fae6c36fa50a)

Reply:
Ok but https://ubuntu.com/certified/vendors/nVidia is already going to https://ubuntu.com/certified/vendors/NVIDIA, so technically it's uncessary?

Reply:
It doesn't seem like it is, the url remains `nVdia`
![image](https://github.com/canonical/ubuntu.com/assets/17607612/7895c6c2-3d01-4c02-9cf4-1e54c878a919)